Mercury-Store/Mercury-Store/Support/Coordinator/HomeTabBarCoordinator.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  HomeTabBarCoordinator.swift
+//  Mercury-Store
+//
+//  Created by mac hub on 02/06/2022.
+//
+
+import Foundation
+import UIKit
+
+
+final class HomeTabBarCoordinator : Coordinator {
+    
+    weak var parentCoordinator: Coordinator?
+    
+    var mainTabBar: UITabBarController!
+    var children: [Coordinator] = []
+    
+    var navigationController: UINavigationController
+    
+    func start() {
+        print("HomeTabBar Coordinator Init")
+        initializeHomeTabBar()
+    }
+    
+    deinit {
+        print("HomeTabBar Coordinator deinit")
+    }
+
+    
+    func initializeHomeTabBar(){
+        
+        let homeNavigationController = UINavigationController()
+        let homeCoordinator = HomeCoordinator.init(navigationController: homeNavigationController)
```

Comment:
```suggestion
        let homeCoordinator = HomeCoordinator(navigationController: homeNavigationController)
```
![Screen Shot 2022-06-02 at 10 22 49 PM](https://user-images.githubusercontent.com/45182214/171731445-012a01e5-01f6-4386-9f2d-d9470f0092b9.png)